src/extensions/common/useFetchCache.ts

```diff
@@ -0,0 +1,103 @@
+import objectHash from 'object-hash'
```

Comment:
Here's the FetchCache! This really simplifies the async fetch flow for timeseries and snippets views